src/config/config.ts

```diff
@@ -385,6 +385,14 @@ const config = convict({
       default: "18.136.36.203",
     },
   },
+  netlify: {
+    accessToken: {
+      doc: "Access token for netlify actions",
+      env: "NETLIFY_ACCESS_TOKEN",
+      format: "required-string",
+      default: "",
```

Comment:
pls give a sane default sir - i think this might fail on cold start

Reply:
We want this to fail if nothing is provided though, so we should leave this blank?

Reply:
eh i derped, forgot this was supposed to be in 1PW/`.env` ._.